src/io/flutter/module/InstallSdkAction.java

```diff
@@ -143,6 +145,7 @@ public void validateSelectedFiles(VirtualFile[] files) throws Exception {
 
       final VirtualFile installTarget = FileChooser.chooseFile(descriptor, null, null);
       if (installTarget != null) {
+        FlutterInitializer.sendAnalyticsAction(getClass().getSimpleName());
```

Comment:
There are 2 actions: git clone only gets called if `git` is found, otherwise we just open a browser on the docs.  This way we get a separate call for each action.